### ISSUE TITLE
More metadata.

### ISSUE THREAD

**Issue by i30817** [libretro/libretro-database#596]
What do you think of expanding this project to have further useful metadata?

It would be better if when the user starts a game without a custom config the setting would be filled in with a 'auto' if the game matches the exact CRC and when saving a new config the default would be saved. 

1. Inherent input latency of a game, for use with the newly added look-ahead latency remover. This would be very useful now and in the future to give the best input user experience.
2. Blacklines removal. There are already user made settings files floating around for this but they have the disadvantage of requiring exact names (and the names are pretty random), exact target resolutions and the user having to go get them. 
3. One that would be useful is to figure out that a game is a 'collection' of isos/floppies. (This can be mostly done with a sorting heuristic and some regex pattern recognition, but i suppose RA would appreciate this being done at the database level instead of UI). It could be used to auto-filll the cd tray on shortcut change, and concatenating the savestate list and auto-changing cds by associating a savestate to a cd.
4. Another related idea would be just for x68k that is actually quite a lot of work and i doubt could be completed soon. X68000 computer emulation often requires two floppies in two different slots (a bit like RAID). If gamers collected 'which' floppies groups are actually needed to play the whole game, you could record them to serve the same functions as the previous paragraph cd collection.

Basically, i think RA should get onboard with enriching the databases, not just merging them with a lookout to fill several settings with the 'best' settings per game. The database seems a natural place to collect this so this info isn't lost / is actually used by normal people.

**Comment by RobLoach** [comment]
As long as they sync up with what's in the DB, then feel free to add them. Having the core inherit settings from the database seems wrong, however. I can understand the benefit, but the DB should not determine how a game should be played, just the information about the game.
